### ISSUE TITLE
screen orientation changes back and forth 

### ISSUE THREAD

**Issue by zilexa** [gabm/TapAndTurn#51]
Just a minor bug on Pixel 1 with Android 9.0.
Tested in WhatsApp: TapAndTurn is running and active. In Android, Auto rotation is enabled. The option "Automatically return to default orientation" is active.

1. While holding your phone vertical, open WhatsApp.
2. Turn the phone horizontal.
3. The icon appears.
4. DO NOT tap the icon, wait for it to disappear. The phone stays in portrait while you are holding it horizontal.
5. Now turn the phone vertical again...

6. EXPECTED: nothing should happen now because the orientation of the screen has never been changed, it is portrait the whole time.

6.. Actual behaviour: at step 5, AFTER going from horizontal back to vertical, I am looking at portrait mode but the orientation suddenly jumps to the orientation jumps to landscape and back to portrait.

When I disable the option "Automatically return to default orientation", this does not happen. But I prefer keeping it on.. 

Just to be clear: it changes the orientation twice at a moment when I am holding my phone vertical. This should never happen: it should only change the orientation once. And only if the current orientation is not the default one.

**Comment by licaon-kter** [comment]
"Automatically return to default orientation" - works as expected... is this a translation issue? The Default orientation is Portrait....hence...it...returns...to...Portrait...

/close as invalid

**Comment by zilexa** [comment]
No, please read again: there is nothing to return to because I did not tap the button!
So the screen orientation is portrait the entire time even when I am holding my phone horizontally. 

Yet when I hold my phone vertical again, TapAndTurn changes the orientation _to landscape_ for 1 second and then reverts it back to portrait. 

It should not change the orientation to landscape. 

I have rewritten my text hopefully it is more clear.

**Comment by licaon-kter** [comment]
Ok, right. Saw this glitch too.

**Comment by gabm** [comment]
@zilexa @licaon-kter is this still happening.. unfortunately I am not able to reproduce!?

**Comment by licaon-kter** [comment]
Yes, it looks like it loses control for a second then regains it. Can't repro at will.

**Comment by zilexa** [comment]
Sorry I don't use your app anymore because on Pixel 1, Android 9.0 the exact feature is now built in! The button to rotate appears in my navigation bar. It works just as your app.

**Comment by rubslopes** [comment]
I was also having this on Oreo 8.1. I solved it by turning the system auto rotation off. Everything else still works fine and now the glitch is gone.
